roman_imsim/skycat.py

```diff
@@ -208,14 +208,10 @@ def getObj(self, index, gsparams=None, rng=None, exptime=30):
         for component in gsobjs:
             if faint:
                 gsobjs[component] = gsobjs[component].evaluateAtWavelength(self.bandpass)
-                gs_obj_list.append(
-                    gsobjs[component] * self._trivial_sed * self.exptime * roman.collecting_area
-                )
+                gs_obj_list.append(gsobjs[component] * self._trivial_sed)
             else:
                 if component in seds:
-                    gs_obj_list.append(
-                        gsobjs[component] * seds[component] * self.exptime * roman.collecting_area
-                    )
+                    gs_obj_list.append(gsobjs[component] * seds[component])
```

Comment:
I didn't see any note about why this is no longer multiplied by `exptime * collecting_area`. It's not immediately obvious to me how any of the changes below addresses this difference.

Reply:
Given that we do `gs_object.withFlux(flux, self.bandpass)` after, it will automatically normalized the SED. So I guess we should either set the flux by normalizing the SED or setting the flux directly but not both.